integration/chat-features.js

```diff
@@ -2,19 +2,46 @@ const faker = require('faker')
 const randomNumber = faker.datatype.number() // generate random number
 const randomMessage = faker.lorem.sentence() // generate random sentence
 
-it('Chat - Send stuff on chat', () => {
-  cy.importAccount()
+describe('Chat Features Tests', () => {
+  it('Chat - Send stuff on chat', () => {
+    cy.importAccount()
 
-  //Validate profile name displayed
-  cy.chatFeaturesProfileName('asdad')
+    //Validate profile name displayed
+    cy.chatFeaturesProfileName('sadad')
 
-  // Click on hamburger menu if width < height
-  cy.get('.toggle-sidebar').should('be.visible').click()
+    // Click on hamburger menu if width < height
+    cy.get('.toggle-sidebar').should('be.visible').click()
 
-  //Validate message and emojis are sent
-  cy.chatFeaturesSendMessage(randomMessage)
-  cy.chatFeaturesSendEmoji('[title="smile"]', '😄')
+    //Validate message and emojis are sent
+    cy.chatFeaturesSendMessage(randomMessage)
+    cy.chatFeaturesSendEmoji('[title="smile"]', '😄')
 
-  //Validate message can be edited
-  cy.chatFeaturesEditMessage(randomMessage, randomNumber)
+    //Validate message can be edited
+    cy.chatFeaturesEditMessage(randomMessage, randomNumber)
+  })
+
+  it('Chat - Verify when clicking on Send Money a quick paypments pop-up appears', () => {
```

Comment:
typo - payments 🔨

Reply:
also, considering atm the quick payments will not appear because is coming soon, maybe we could rename the test to ''Chat - Verify when clicking on Send Money coming soon appears'' or something similar